### PR TITLE
31437: Whenever an enrollment is serialized from GlueDB, include employer contact information

### PR DESCRIPTION
**Pull request by zacharylandes**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [31437](https://devops.dchbx.org/redmine/issues/31437) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-Disqueued |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | @TreyE |

### CODE REVIEW

**Review by TreyE — CHANGES_REQUESTED** [review]
Check more carefully here.  You don't want to write out empty xml tags.

**Review by bill-transue — CHANGES_REQUESTED** [review]
Please update the matching spec, `spec/views/enrollment_events/policy.xml.haml_spec.rb`. Thanks!

File: app/views/enrollment_events/_policy.xml.haml [hunk 1]
```diff
@@ -72,6 +72,8 @@
         - if !employer.dba.blank?
           %dba= employer.dba
         %fein= employer.fein
+        = render :partial => "enrollment_events/employer_with_contacts", :object => employer
```
Comment (bill-transue) [inline app/views/enrollment_events/_policy.xml.haml:75]: Good job breaking these out in to partials.

**Review by bill-transue — APPROVED** [review]
This extends the `app/views/enrollment_events/_policy.xml.haml` view with two new partials, `enrollment_events/employer_with_contacts` and `enrollment_events/employer_with_office_locations`. The original policy view spec is updated to test these partials as well as new specs for the individual partials. Updates were made to the `ActionPublishHelper` to make use of the new view extensions and so its spec was updated as well. Good job!

**Review by TreyE — APPROVED** [review]

**Comment by TreyE** [comment]
This has already been merged and released under a different PR.

**Pull request closed without merging**